module/epochs/epoch_lookup_test.go

```diff
@@ -171,38 +160,37 @@ func (suite *EpochLookupSuite) TestEpochForViewWithFallback_EpochFallbackTrigger
 // in the protocol state.
 func (suite *EpochLookupSuite) TestProtocolEvents_EpochExtended() {
 	// previous and current epochs will be committed
-	suite.CommitEpochs(suite.prevEpoch)
-	epoch, extension := suite.epochFixtureWithExtension(suite.currEpoch.counter, suite.currEpoch.firstView, suite.currEpoch.finalView+1000)
-	suite.epochQuery.Add(epoch)
-
-	epochs := []epochRange{
-		suite.prevEpoch,
-		{counter: suite.currEpoch.counter, firstView: suite.currEpoch.firstView, finalView: extension.FinalView},
-	}
-
-	header := unittest.BlockHeaderFixture()
-	suite.state.On("AtHeight", header.Height).Return(suite.snapshot).Times(4)
+	epochs := []epochRange{suite.prevEpoch, suite.currEpoch}
+	suite.CommitEpochs(suite.prevEpoch, suite.currEpoch)
 
 	suite.CreateAndStartEpochLookup()
 
-	suite.lookup.EpochExtended(suite.currEpoch.counter, header, extension)
+	extension := flow.EpochExtension{
+		FirstView: suite.currEpoch.firstView + 1,
+		FinalView: suite.currEpoch.finalView + 100,
+	}
+	suite.lookup.EpochExtended(suite.currEpoch.counter, nil, extension)
 
 	// wait for the protocol event to be processed (async)
 	assert.Eventually(suite.T(), func() bool {
-		_, err := suite.lookup.EpochForViewWithFallback(extension.FinalView)
+		_, err := suite.lookup.EpochForView(extension.FinalView)
 		return err == nil
 	}, 5*time.Second, 50*time.Millisecond)
 
 	// validate queries are answered correctly
```

Comment:
```suggestion
	// validate queries are answered correctly
	suite.currEpoch.finalView = extension.FinalView // expect final view to have been updated from extension
```
To test the same view range as we expect the EpochLookup component to have